Add batch page

Sreelatha-Vallabhaneni opened the pull request:
I created an endpoint to get id of the user by UID, which is needed to create the batch.

gizemcandemir commented:
Hi @zkwsk, it worked with me :thinking_face: I mean I can see the page `http://localhost:3000/addbatch` when I'm logged in with ningry0@csmonitor.com.

gizemcandemir commented on src/client/containers/add-batch-page/add-batch-page.component.js:
```diff
@@ -1,18 +1,19 @@
     <>
-      <div>
+      <div className="batch-page-nav">
         <SidebarMenu isActive={true} isUser={true} />
```
`isActive` props should be `false` for Add Batch page (it's only true for Chart Detail Page).

gizemcandemir commented on src/client/App.js:
```diff
@@ -39,6 +40,7 @@ function App() {
             component={ForgotPassword}
           />
           <PrivateRoute exact path="/dashboard" component={Dashboard} />
+          <PrivateRoute exact path="/addbatch" component={AddBatch} />
```
I would write the path as `"/add-batch"` with the dash between the words.

dpfernandes replied:
Yes please use add-batch.

It's already working like that on the side bar menu

![image](https://user-images.githubusercontent.com/6642037/80010727-1f810c80-84cb-11ea-98ea-317277c39092.png)

Sreelatha-Vallabhaneni commented:
Hi @zkwsk  I fixed that 401 issue and other issues which were reviewed by @gizemcandemir  (i fixed all the issues) but i have a git problem it won't allow me to push it. i tried in many ways and @cecastosic also tried to help ....it didn't work out.......plz help me to fix it

Sreelatha-Vallabhaneni commented:
I am closing this PR #198 because I have created a new PR #207 for ADD BATCH
